src/main/java/cd/go/contrib/elasticagents/dockerswarm/elasticagent/validator/DockerSecretValidator.java

```diff
@@ -42,17 +42,8 @@ public DockerSecretValidator(PluginRequest pluginRequest) {
     @Override
     public ValidationResult validate(Map<String, String> elasticProfile) {
         final ValidationResult validationResult = new ValidationResult();
-
         try {
             final DockerSecrets dockerSecrets = DockerSecrets.fromString(elasticProfile.get("Secrets"));
-
-            if (!dockerSecrets.isEmpty()) {
```

Comment:
Removing this validation will cause create agent request to fail.

Instead of removing, can we move this validation to `create-agent` call to fail at runtime and add an appropriate server health message for the failure?